app/javascript/components/FacetsPanel.js

```diff
@@ -1,20 +1,20 @@
 import React, { useState, useEffect, useContext } from 'react';
 import FacetControl from './FacetControl';
 import MoreFacetsButton from './MoreFacetsButton';
-import { StudySearchContext } from 'components/search/StudySearchProvider'
+import { SearchFacetContext } from 'components/search/SearchFacetProvider'
```

Comment:
Imports that rely on Rails `webpacker.yml` settings can't take advantage of standard IDE settings out of the box, e.g. showing docstrings for imported functions or classes, `Look up definition`, and `Rename symbol`.  Relative imports do, so I suggest:

```suggestion
import { SearchFacetContext } from './search/SearchFacetProvider'
```

Here and for all other imports.